numexpr/tests/test_numexpr.py

```diff
@@ -296,6 +296,17 @@ def test_str_contains_long_needle(self):
         res = evaluate('contains(a, b)')
         assert_equal(res, True)
 
+    def test_if_constant_bool(self):
+        a = True
+        b = array([1, 2])
+        c = array([3, 4])
+        res = evaluate('if(a, b, c)')
```

Comment:
numexpr uses where instead of if (if -> where translation is done by LIAM2 -- I yielded to popular demand when I probably shouldn't have)